server-common/src/main/java/org/apache/kafka/server/network/EndpointReadyFutures.java

```diff
@@ -79,17 +80,17 @@ public Builder addReadinessFutures(
         /**
          * Build the EndpointReadyFutures object.
          *
-         * @param authorizer    The authorizer to use, if any. Will be started.
-         * @param info          Server information to be passed to the authorizer.
+         * @param authorizerPlugin    The authorizer to use, if any. Will be started.
```

Comment:
How about update to `The authorizer which is wrapped in a {@link org.apache.kafka.common.internals.Plugin} to use`?